app/packages/embeddings/src/tracesToData.tsx

```diff
@@ -70,3 +73,48 @@ export function tracesToData(
       };
     });
 }
+
+const getLabelColor = (key: string, setting: CustomizeColor) => {
```

Comment:
`getLabelColor` and `getConvertedColor` types and implementation seem to differ. Could lead to bugs